psake-build-helpers.ps1

```diff
@@ -0,0 +1,38 @@
+function Set-Project-Properties($version) {
+    write-host "$version"
+
+    set-regenerated-file "$pwd/Directory.Build.props" @"
+<Project>
+    <PropertyGroup>
+        <Version>$version</Version>
+    </PropertyGroup>
+</Project>
+"@
+}
+
+function Get-Copyright {
```

Comment:
Can we remove this method?

Reply:
I would recommend using it to update the copyright message in the metadata. If you continue to use the `Directory.Build.props` file, you can add a `<Copyright>` tag and include the result of the `Get-Copyright()` function to apply the copyright message to the DLLs and to the package itself.